FrameworkSystem/DB/ProxyDB.py

```diff
@@ -3,24 +3,27 @@
 
 __RCSID__ = "$Id$"
 
+import os
+import glob
 import time
+import urllib
 import random
-import types
 import hashlib
-import urllib
+import commands
 
 from DIRAC import gConfig, gLogger, S_OK, S_ERROR
-from DIRAC.Core.Utilities import DErrno
 from DIRAC.Core.Base.DB import DB
-
+from DIRAC.Core.Utilities import DErrno
+from DIRAC.Core.Security import Properties
+from DIRAC.Core.Security.VOMS import VOMS
+from DIRAC.Core.Security.MyProxy import MyProxy
 from DIRAC.Core.Security.X509Request import X509Request  # pylint: disable=import-error
 from DIRAC.Core.Security.X509Chain import X509Chain, isPUSPdn  # pylint: disable=import-error
-from DIRAC.Core.Security.MyProxy import MyProxy
-from DIRAC.Core.Security.VOMS import VOMS
-from DIRAC.Core.Security import Properties
-from DIRAC.ConfigurationSystem.Client.Helpers import Registry
+from DIRAC.ConfigurationSystem.Client.Helpers import Registry, Resources
 from DIRAC.ConfigurationSystem.Client.PathFinder import getDatabaseSection
 from DIRAC.FrameworkSystem.Client.NotificationClient import NotificationClient
+# pylint: disable=import-error,no-name-in-module
```

Comment:
what for do you need this ?

Reply:
Now when this PR merged with PR with proxy providers, its not need. Fixed